dns/bind/src/opnsense/mvc/app/models/OPNsense/Bind/Record.xml

```diff
@@ -40,6 +40,7 @@
                         <RP>RP</RP>
                         <RRSIG>RRSIG</RRSIG>
                         <SRV>SRV</SRV>
+                        <SRV>SSHFP</SRV>
```

Comment:
The XML tag name is the key so it can't be "SRV" again.

Reply:
(looks a bit silly, the text in the tag is the human readable and translatable label. In this case it's the same.

Reply:
Well, I clearly need another bucket of ☕